tests/sentry/api/endpoints/test_project_templates_index.py

```diff
@@ -55,3 +58,70 @@ def test_get__no_access(self):
         # Ensure this errors with 403, as the user does not have access to the organization
         response = self.get_error_response(org_two.id, status_code=403)
         assert response.status_code == 403
+
+
+class ProjectTemplateIndexPostTest(APITestCase):
+    endpoint = "sentry-api-0-organization-project-templates"
+
+    def setUp(self):
+        super().setUp()
+        self.user = self.create_user()
+        self.org = self.create_organization(owner=self.user)
+        self.team = self.create_team(organization=self.org, members=[self.user])
```

Comment:
nit: there's properties (on the base fixtures I think) that create these automatically for you

Reply:
fyi - this didnt work for me, looking a little closer at the helpers it's cause `self.org` vs `self.organization` and the create_team helper doesn't add any members. i'll update the name for `org` => `organization` but `members` is correct.

Reply:
I took a quick look at the `self.team` property, is looks like `self.user` should be included as a team member b/c it the user should be a part of `self.organization`.
https://github.com/getsentry/sentry/blob/dcec0930c33979f88929787fc1b43badf89f0df6/src/sentry/testutils/fixtures.py#L66-L71

Reply:
_shakes fist_ no fair!! the user wasn't part of the org when i left the comment! it changed when i created the base class for `setUp` for the comment colleen left.